modules/reference/services/proxy-server/3proxy-config.nix

```diff
@@ -9,14 +9,26 @@
 let
   cfg = config.ghaf.reference.services.proxy-server;
   inherit (lib) mkEnableOption mkIf;
-
   proxyUserName = "proxy-user";
   proxyGroupName = "proxy-admin";
-  proxyAllowListName = "allowlist.txt";
-  proxyWritableAllowListPath = "/etc/${proxyAllowListName}";
-  ms-url-fetcher = pkgs.callPackage ./ms_url_fetcher.nix {
-    allowListPath = proxyWritableAllowListPath;
-  };
+  url-fetcher = pkgs.callPackage ./url_fetcher.nix { };
+
+  msUrls = "https://endpoints.office.com/endpoints/worldwide?clientrequestid=b10c5ed1-bad1-445f-b386-b919946339a7";
+  ssrcUrls = "https://raw.githubusercontent.com/tiiuae/ghaf-rt-config/refs/heads/additional-urls/network/urls/ssrc_urls.json";
+  tiiUrls = "https://raw.githubusercontent.com/tiiuae/ghaf-rt-config/refs/heads/additional-urls/network/urls/tii_urls.json";
+  msExtraUrls = "https://raw.githubusercontent.com/tiiuae/ghaf-rt-config/refs/heads/additional-urls/network/urls/ms_extra_urls.json";
+
+  msAllowFilePath = "3proxy/ms_whitelist.txt";
+  ssrcAllowFilePath = "3proxy/ssrc_whitelist.txt";
+  tiiAllowFilePath = "3proxy/tii_whitelist.txt";
```

Comment:
Instead of hardcoding TII/SSRC, could we make this generic?

Reply:
Should we store those in the same JSON file?